lib/screens/results/model/results_model.dart

```diff
@@ -0,0 +1,54 @@
+class Result {
```

Comment:
Should we have the respective WoD id here? So we can have a way to link this result to the respective WoD?